lib/Notes/NoteForm/NoteView.js

```diff
@@ -234,10 +240,12 @@ export default class NoteView extends Component {
                 open={assigned}
                 onToggle={this.handleSectionToggle}
               >
-                <ReferredRecord
-                  referredEntityData={referredEntityData}
-                  entityTypeTranslationKeys={entityTypeTranslationKeys}
-                />
+                { referredEntityData &&
+                  <ReferredRecord
+                    referredEntityData={referredEntityData}
+                    entityTypeTranslationKeys={entityTypeTranslationKeys}
```

Comment:
please move it to render method for consistency with NoteForm component https://github.com/folio-org/stripes-smart-components/blob/master/lib/Notes/NoteForm/NoteForm.js#L132